automation/taskcluster/decision_task.py

```diff
@@ -10,7 +10,10 @@
 
 def main(task_for):
     if task_for == "github-pull-request":
-        android_linux_x86_64()
+        if "[full-ci]" in os.environ["GITHUB_PR_TITLE"]:
+            android_multiarch()
```

Comment:
It might be worth changing this to not build nonmegazords in release mode (for non-releases) as in: https://github.com/mozilla/application-services/blob/1d1d7af65ae5fcf41ce73a412e22a4907f938457/automation/taskcluster/decision_task.py#L129

The benefit of this is getting `debug_assert`s and such in the code, in addition to a much faster build.